Docs status updates - Fixes #150 (#268)
nautobot/nautobot: nautobot/docs/models/ipam/prefix.md

Prefixes are automatically organized by their parent aggregates. Additionally, each prefix can be assigned to a particular site and virtual routing and forwarding instance (VRF). Each VRF represents a separate IP space or routing table. All prefixes not assigned to a VRF are considered to be in the "global" table.

> Each prefix must be assigned a status and can optionally be assigned a role. These terms are often used interchangeably so it's important to recognize the difference between them. The **status** defines a prefix's operational state. The following default statuses are provided, but may be customized as desired (more information on custom status model [`here.`](https://nautobot.readthedocs.io/en/latest/models/extras/status/)):

Reviewer: ```suggestion
Each prefix must be assigned a [status](https://nautobot.readthedocs.io/en/latest/models/extras/status/) and can optionally be assigned a role. These terms are often used interchangeably so it's important to recognize the difference between them. The **status** defines a prefix's operational state. The following statuses are available by default:
```
